tests/perf/v2/test_batch_sequential_v2.py

```
@@ -0,0 +1,149 @@
+    Make a single API request and return the response data and elapsed time.
+    Raises exception if the request fails.
+    """
+    start_time = time.time()
+    try:
+        response = await client.post(url, json=payload)
+        elapsed = time.time() - start_time
+
+        if response.status_code != 200:
+            raise Exception(
+                f"Request {request_number} failed with status {response.status_code}: {response.text}"
+            )
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)